Description: drop MacOS, iOS, and Windows dependencies
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

Avoid trying to build for MacOS, iOS, or Windows.  This is for
building on debian, for debian.

Forwarded: not-needed

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,23 +54,9 @@
 unix_sysv = []
 use_std = ["libc/use_std"]
 
-[target."cfg(all(any(target_os = \"macos\", target_os = \"ios\"), not(feature = \"unix_sysv\")))".dependencies.mach2]
-version = "0.4.1"
-default-features = false
-
 [target."cfg(any(unix, target_os = \"dragonfly\"))".dependencies.libc]
 version = "0.2"
 default-features = false
 
-[target."cfg(target_os = \"windows\")".dependencies.winapi]
-version = "0.3.*"
-features = [
-    "memoryapi",
-    "handleapi",
-    "sysinfoapi",
-    "winbase",
-]
-default-features = false
-
 [badges.maintenance]
 status = "passively-maintained"
